app/views/users/index.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Here is a list of all users</h1>
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <body>
+<div>
+<% @users.each do |user| %>
+<section class="rounded-lg p-8 flex items-center">
+    <img class="border" src="<%= user.photo %>">
+    <div class="ml-4 text-center flex inform border">
+      <h2 class="font-bold mr-4 mb-2 header-font"><%= user.name %></h2>
+      <p class="text-red-600 par">Number of posts: <%= user.posts_counter %></p>
+    </div>
+    <!-- display user's posts, comments, likes, etc. -->
```

Comment:
- [ ] Please Clicking on each user should take the user to their show page , currently there is no link in the user index page for each user please fix this , so the users can navigate easly through your app.